### PR TITLE
fixing the AdultData LoadColumn, and IrisWithAllFeatures class.

### PR DESCRIPTION
**Pull request by sfilipi**
Fixing errors regarding data loading,  spotted in the cookbook.

### CODE REVIEW

File: docs/code/MlNetCookBook.md [hunk 1]
```diff
@@ -285,11 +285,11 @@ Or by creating a data model for it:
 ```csharp
 private class AdultData
 {
-    [LoadColumn("0", "10"), ColumnName("Features")]
-    public float FeatureVector { get; }
+    [LoadColumn(0, 10), ColumnName("Features")]
+    public float FeatureVector { get; set;}
```
Comment (wschin) [inline docs/code/MlNetCookBook.md]: Maybe just
```suggestion
    public float FeatureVector;
```
?

File: docs/code/MlNetCookBook.md [hunk 2]
```diff
@@ -285,11 +285,11 @@ Or by creating a data model for it:
...
 
     [LoadColumn(11)]
-    public float Target { get; }
+    public float Target { get; set;}
```
Comment (wschin) [inline docs/code/MlNetCookBook.md]: Same here. Not sure if having `get` and `set` is a convention but I'd remove them for shorter bug equivalent code.
```suggestion
    public float Target;
```

File: docs/code/MlNetCookBook.md (hunk 1 above)
Comment (sfilipi) [reply docs/code/MlNetCookBook.md]: I'll double-check. I believe it has to be a property, not a field.

Comment (shauheen) [inline docs/code/MlNetCookBook.md]: >float [](start = 11, length = 5)

I don't think this would work.

**Comment by sfilipi** [comment]
merging this to make the doc at pair with the code in the tests, and taking on another pr to refactor the cookbook and verify the examples

**Pull request merged**